runtime/msgpack-serialization.h

```diff
@@ -70,6 +70,20 @@ inline Optional<string> f$instance_serialize(const class_instance<InstanceClass>
   return result;
 }
 
+template<class InstanceClass>
+inline string f$instance_serialize_safe(const class_instance<InstanceClass> &instance) noexcept {
```

Comment:
I think it should return `Optional<string>` too: `nullopt` when our exception is 'thrown'